site/v1.1/README.md

```diff
@@ -167,7 +167,7 @@ For information about how to shut down a domain running in Kubernetes, see [Shut
 
 To permanently remove the Kubernetes resources for a domain from a Kubernetes cluster, run the [Delete WebLogic domain resources](/kubernetes/delete-weblogic-domain-resources.sh) script. This script will delete a specific domain, or all domains, and all the Kubernetes resources associated with a set of given domains. The script will also attempt a clean shutdown of a domain’s WebLogic pods before deleting its resources.  You can run the script in a test mode to show what would be shutdown and deleted without actually performing the shutdowns and deletions.   For script help, use its `-h` option.
 
-The script will remove only domain-related resources which are labeled with the `domainUID` label, such as resources created by the [Create WebLogic domain](/kubernetes/create-weblogic-domain.sh) script or the [integration tests](/src/integration-tests/bash/run.sh).  If you manually created resources and have not labelled them with a `domainUID`, the script will not remove them.   One way to label a resource that has already been deployed is:
```

Comment:
I don't think we should remove the references to run.sh from V1.1 doc because run.sh is only removed in V2.0.

Reply:
A question: why release V1.1 doc in 2.0 source code?

Reply:
@markxnelson @rjeberhard I don't remember the details about this. can you also confirm if my understanding/comments are correct?

Reply:
The 1.1 docs were preserved so you could see the documentation for that release.  In general, I agree that we should not change the historical documents.